draft-tuexen-tsvwg-rfc4960-errata.xml

```diff
@@ -733,6 +733,74 @@ partial_bytes_acked and cwnd in the congestion avoidance phase.</t>
 </section>
 </section>
 
+<section title='HEARTBEAT ACK and the association error counter'>
+<section title='Description of the Problem'>
+<t>Section 8.1 and Section 8.3 of <xref target="RFC4960"/>
+prescribe that the receiver of a HEARTBEAT ACK must reset the
+association overall error counter. In some circumstances, e.g. when
+a router discards DATA chunks but not HEARTBEAT chunks due to the
+larger size of the DATA chunk, it is better to not clear the
+association error counter by the HEARTBEAT ACK and reset it by the
+SACK only.</t>
+</section>
```

Comment:
On Oct 20, 2015, at 2:14 PM, Maxim <notifications@github.com> wrote:

> In draft-tuexen-tsvwg-rfc4960-errata.xml:
> 
> > @@ -733,6 +733,74 @@ partial_bytes_acked and cwnd in the congestion avoidance phase.</t>
> >  </section>
> >  </section>
> >  
> > +<section title='HEARTBEAT ACK and the association error counter'>
> > +<section title='Description of the Problem'>
> > +<t>Section 8.1 and Section 8.3 of <xref target="RFC4960"/>
> > +prescribe that the receiver of a HEARTBEAT ACK must reset the
> > +association overall error counter. In some circumstances, e.g. when
> > +a router discards DATA chunks but not HEARTBEAT chunks due to the
> > +larger size of the DATA chunk, it is better to not clear the
> > +association error counter by the HEARTBEAT ACK and reset it by the
> > +SACK only.</t>
> > +</section>
> do you propose to add it or to replace "In some circumstances, e.g. when a router discards DATA chunks but not HEARTBEAT chunks due to the larger size of the DATA chunk, it is better to not clear the association error counter by the HEARTBEAT ACK and reset it by the SACK only."
> ?
> 
> 
Why?

That is exactly how one would implement PMTU discovery with the PAD chunk in combination.
I.e. we would send HB’s padded to probe for size.. 

If you go to far you would find this.. and in theory you will either discover it
via a ICMP or worse case is you can back down the MTU size through other
logic (if you believe you won’t receive ICMP).

R

> —
> Reply to this email directly or view it on GitHub.
> 

--------
Randall Stewart
rrs@netflix.com
803-317-4952